meeting_minutes/231207_SARIF_TC_82.md

```diff
@@ -0,0 +1,161 @@
+| Tim        | Hudson    | Cryptsoft Pty Ltd.      | Member                  | No      |
+| Vadim      | Okun      | NIST                    | Observer                | No      |
+| Yekaterina | O'Neil    | Micro Focus             | Voting member           | Yes     |
+| Zvika      | Avivi     | Micro Focus             | Observer                | No      |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* [https://www.oasis-open.org/committees/download.php/71546/agenda_20231207.html](https://www.oasis-open.org/committees/document.php?document_id=71546&wg_abbrev=sarif)
+
+The agenda was approved.
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2023-11-09 Meeting #81](https://www.oasis-open.org/committees/document.php?document_id=71488&wg_abbrev=sarif)
+
+The minutes were approved.
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on Charles to provide extensions to the roadmap document
+  [Future.md](https://github.com/oasis-tcs/sarif-spec/blob/main/Future.md) per pull request
+  * ONGOING
+* ACTION on Co-Chair David to investigate OASIS mechanisms for fast-tracking simple transformation of existing spec to markdown
+  * DONE
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective voting members attending their first meeting
+
+* Stacy Wray
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair David)
+
+- Scheduled Teleconference (Thursday at 08:00 PT / **16:00** UTC for 1.5 hours)
+    ```
+    January 11, 2024
+    ```
+- Proposed Teleconference (Thursday at 08:00 PT / **16:00** UTC for 1.5 hours)
+    ```
+    February 8, 2024
+    ```
+
+# 3. Discussion
+
+## 3.1 Review current state of ecosystem ongoing work
+
+### 3.1.1 Related activities (OPENSSF, etc.)
+
+* No news
+
+### 3.1.2 Other Ecosystem Items
+
+* David Malcolm: GCC Diagnostic library now has a wiki page at <https://gcc.gnu.org/wiki/libdiagnostics>
+
+## 3.2 Review outcomes of subgroup discussions
+
+* No news
+
+## 3.3 Discuss the list of small non-breaking changes for SARIF v2.2
+
+ * Stefan: Explains his proposal for going forward technically per pull 
+           request [Seeding the SARIF v2.2 workplace - separating source and deliverables #599](https://github.com/oasis-tcs/sarif-spec/pull/599).
+ * All discuss
+ * David: Suggests that 
+   * at the January, 11 meeting the TC has the reference representation of v2.1 Errata 01 to approve it
+   * then, by the February, 8 meeting to have approved proposals to merge into the v2.2 working draft
```

Comment:
- Add comma after "meeting"
- Add "ready" before "to merge"
- Add period at end of sentence.